src/stories/Library/nav-teaser/NavTeaser.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from "react";
+
+type NavTeaserProps = {
+  title: string;
+  subtitle?: string;
+};
+
+const NavTeaser: FC<NavTeaserProps> = ({ title, subtitle }) => {
+  return (
+    <article className="nav-teaser">
```

Comment:
Was it decided to remove the pagefold on hover? 

![image](https://github.com/danskernesdigitalebibliotek/dpl-design-system/assets/13272656/23a6fefd-ee5c-4ec8-a613-b64897543b72)

Reply:
I see yea